angular/src/app/features/films/film/film.component.html

```diff
@@ -0,0 +1,3 @@
+<td>{{ film?.title }}</td>
+<td>{{ film?.director }}</td>
+<td>{{ film?.created | date:'d.LL.YYYY'}}</td>
```

Comment:
I don't see optional properties in your model. I think you can rewrite it.
```suggestion
<ng-container *ngIf="film !== null">
  <td>{{ film.title }}</td>
  <td>{{ film.director }}</td>
  <td>{{ film.created | date:'d.LL.YYYY'}}</td>
</ng-container>
```